### DISCUSSION TITLE
Mark scrolled entries as read

### DISCUSSION THREAD

**Discussion by Roterfux** [jocmp/capyreader#186] [Feature Requests]
This would be a nice to have. Since not all feed entries are always interesting and I tend to scroll though my feed, this feature would be a nice option to have.

**Comment by jocmp** [comment]
@Roterfux I've seen this as an option in other feed readers before but I've never personally used it. Is there an app you've used that has done it well? I'll take a look.

**Reply by ghost** [reply]
Feedly has a similar feature.

**Comment by Roterfux** [comment]
Hey @jocmp, 
I'm currently using https://github.com/bubelov/news. And they've done it well. At least it works for me.
br

**Comment by jocmp** [comment]
There's a few other features I'm working on, but this is on the list. You can track progress here https://github.com/jocmp/capyreader/issues/271.
